cmd/promtool/main.go

```diff
@@ -353,8 +353,10 @@ func checkConfig(agentMode bool, filename string) ([]string, error) {
 	}
 
 	for _, scfg := range cfg.ScrapeConfigs {
-		if err := checkFileExists(scfg.HTTPClientConfig.BearerTokenFile); err != nil {
-			return nil, errors.Wrapf(err, "error checking bearer token file %q", scfg.HTTPClientConfig.BearerTokenFile)
+		if scfg.HTTPClientConfig.Authorization != nil {
+			if err := checkFileExists(scfg.HTTPClientConfig.Authorization.CredentialsFile); err != nil {
+				return nil, errors.Wrapf(err, "error checking authorization credentials file %q", scfg.HTTPClientConfig.Authorization.CredentialsFile)
```

Comment:
This could also be the bearer token file, so we should specify that in the error message to avoid confusion.

```suggestion
				return nil, errors.Wrapf(err, "error checking authorization credentials or bearer token file %q", scfg.HTTPClientConfig.Authorization.CredentialsFile)
```

Reply:
Ah yes, bearer token files are still accepted. Updated the message.